programmers-coding-test/sangjun/greedy/큰수의법칙.js

```diff
@@ -0,0 +1,34 @@
+function largeNumRule(n, m, k) {
+  const randomNum = [];
+
+  let result = 0;
+
+  for (let i = 0; i < n; i++) {
+    randomNum.push(Math.floor(Math.random() * 9 + 1));
+  }
+
+  let descendNum = randomNum.sort();
```

Comment:
일반 sort()를 이용하면 js는 아스키코드로 정렬한다. 문자로 정렬한다는 말.
숫자로 정렬하도록 파라미터로 알려줘야함.

```suggestion
  const descendNum = randomNum.sort((a, b) => b - a); // 내림차순으로 정렬하여 0번지 값을 가져오는게 가독성에 좋다. 효율도 더 좋음.
```